instances/service/KGSearchUI.jsonld

```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/service/KGSearchUI",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Service",
+  "definition": "Web application for searching the EBRAINS Knowledge Graph",
```

Comment:
"The 'EBRAINS Knowledge Graph Search User Interface', short KGS UI, is a web application for searching the EBRAINS Knowledge Graph."

Reply:
changed accordingly